library/src/main/java/pl/rspective/voucherify/android/client/api/VoucherifyApi.java

```diff
@@ -22,7 +23,7 @@
      * @return
      */
     @GET("/client/v1/validate")
-    VoucherResponse validateVoucher(@QueryMap Map<String, String> queryParams);
+    Observable<VoucherResponse> validateVoucher(@QueryMap Map<String, String> queryParams);
```

Comment:
Why we force users to use Observables ? What if someone doesn't want to use RxJava/Android world, what if someone what to use a normal asynchronous call? I would rather think about 'Call' - https://square.github.io/retrofit/2.x/retrofit/retrofit2/Call.html

The current version of the SDK allows the users to call API in the synchronous, asynchronous or rx way.